rust/src/main.rs

```diff
@@ -65,6 +67,57 @@ async fn inner_async_main(args: Vec<String>) -> Result<i32> {
     .await?
 }
 
+/// Multicall entrypoint for `ostree-container`.
+async fn multicall_ostree_container(args: Vec<String>) -> Result<i32> {
+    ostree_ext::cli::run_from_iter(args).await?;
+    Ok(0)
+}
+
+/// Dispatch multicall binary to relevant logic, based on callname from `argv[0]`.
+async fn dispatch_multicall(callname: String, args: Vec<String>) -> Result<i32> {
+    match callname.as_str() {
+        // TODO(lucab): fix the final callname for this after ostree side at
+        // https://github.com/ostreedev/ostree/issues/2480 is settled.
+        "ostree-container" => multicall_ostree_container(args).await,
```

Comment:
Had some thoughts on this here https://github.com/ostreedev/ostree/issues/2480#issuecomment-996239587